packages/gravity/kube-testing/src/spec/echo.ts

```diff
@@ -67,6 +67,25 @@ export class EchoTestPlan implements TestPlan<EchoTestSpec, EchoAgentConfig> {
   space!: Space;
   spaceKey?: PublicKey;
 
+  defaultSpec(): EchoTestSpec {
```

Comment:
maybe it makes sense to move default specs to a default yaml config files for each testbench?